scripts/install.sh

```diff
@@ -15,5 +15,8 @@ echo $ENABLE_CONTRIB > contrib.enabled
 echo $ENABLE_HEADLESS > headless.enabled
 echo $ENABLE_ROLLING > rolling.enabled
 set -x
+if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+  pip install --upgrade pip
+fi
```

Comment:
Let's move it to `travis_config.sh` somewhere [here](https://github.com/asmorkalov/opencv-python/blob/as/pre-4.7.0/travis_config.sh#L147). In case the issue with pip only with Python 3.7.
```
created virtual environment CPython3.7.13.final.0-64 in 452ms
  creator CPython3Posix(dest=/venv, clear=False, no_vcs_ignore=False, global=False)
  seeder FromAppData(download=False, wheel=bundle, pip=bundle, setuptools=bundle, via=copy, app_data_dir=/root/.local/share/virtualenv)
    added seed packages: pip==19.1.1, setuptools==43.0.0, wheel==0.33.6
```

Reply:
It works well in the current status of `4.x` branch:
```
created virtual environment CPython3.7.15.final.0-64 in 871ms
  creator CPython3Posix(dest=/venv, clear=False, no_vcs_ignore=False, global=False)
  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/root/.local/share/virtualenv)
    added seed packages: pip==22.3.1, setuptools==65.5.1, wheel==0.38.4
```

The main reason - the docker image for tests is different. Idk why it has changed in this PR, because you updates multibuild submodule and it shouldn't happen.

The wrong image (this had been using for some time and was dropped half a year ago) - `matthewbrett/trusty:64`

The proper image - `multibuild/focal_x86_64`

This rule should work here, but it doesn't ([link](https://github.com/multi-build/multibuild/blob/devel/travis_linux_steps.sh#L131))
```
local docker_image="multibuild/focal_{PLAT}"
```

**TL;DR**
While I was writing this comment, I found out that commit for multibuild submodule is too old (2020 year). Move back to the previous version which was stable or update to the latest one (it also can solve Python 3.11 issues).